content/writing_tests/interact.md

```diff
@@ -7,7 +7,84 @@ weight : 3
 # Interacting with the environment
 
 In addition to screenshot-based test actions, Sakuli is capable of utilizing your keyboard, clipboard and environment variables.
-All these features are incorporated in the `Environment` class.
+All these features are incorporated with the `ThenableEnvironment` interface.
```

Comment:
We should add a link to https://sakuli.io/apidoc/sakuli-legacy/interfaces/thenableenvironment.html here